widen-media.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name:       Widen Media
  * Description:       Search and add Widen digital assets to your WordPress media library.
- * Version:           2.6.5
+ * Version:           2.6.7
```

Comment:
Any reason we are skipping 2.6.6?

Reply:
I missed it in the last PR. So trying to match the tag version number which will be 2.6.7 next.